model/task/task.go

```diff
@@ -1150,7 +1150,7 @@ func (t *Task) SetHasCedarResults(hasCedarResults, failedResults bool) error {
 		return err
 	}
 
-	if !t.DisplayOnly && t.IsPartOfDisplay() {
```

Comment:
It looks like isPartOfDisplay checks if `t.DisplayTaskId == nil`, not if it's "". So it looks like we'd still need `!t.DisplayOnly`.

Reply:
Checking !t.DisplayOnly here is really just a shortcut; IsPartOfDisplay will always be false. That being said, with this change any new display task will have t.DisplayTaskId != nil (because it will be set to "") so we'll still shortcircuit

Reply:
I may be misunderstanding your comment, but "" _is_ not nil, so we would correctly be skipping the if statement in IsPartOfDisplay and check the `utility.FromStringPtr(t.DisplayTaskId) != ""` line directly

Reply:
Ah, you're right, I was misreading it.